po/ru.po

```diff
@@ -0,0 +1,2471 @@
+"process to set correct owner and group information."
+msgstr ""
+"Убедитесь, что смена владельца ожидаемая. Если нет, настройте процесс сборки "
+"пакета, чтобы указать правильную информацию о владельце и группе."
+
+#: lib/results.h:124
+msgid "Consult the shell documentation for proper syntax."
+msgstr "Обратитесь к документации оболочки shell для правки синтаксиса."
+
+#: lib/results.h:125
+msgid ""
+"The file referenced was not a known shell script in the before build but is "
+"now a shell script in the after build."
+msgstr ""
+"Указанный файл не был известным сценарием оболочки shell в предыдущей "
+"сборке, но теперь является сценарием оболочки shell в последующей сборке."
+
+#: lib/results.h:126
+msgid ""
+"The referenced shell script is invalid. Consider debugging it with the '-n' "
+"option on the shell to find and fix the problem."
+msgstr ""
+"Указанный shell скрипт недействителен. Рассмотрите возможность отладки с "
+"помощью опции '-n' в терминале, чтобы найти и исправить проблему."
+
+#: lib/results.h:129
+msgid "See annocheck(1) for more information."
+msgstr "См. Дополнительную информацию в annocheck (1)."
+
+#: lib/results.h:132
+msgid ""
+"DT_NEEDED symbols have been added or removed.  This happens when the build "
+"environment has different versions of the required libraries.  Sometimes "
+"this is deliberate but sometimes not.  Verify these changes are expected.  "
+"If they are not, modify the package spec file to ensure the build links with "
+"the correct shared libraries."
+msgstr ""
+"Добавлены или удалены символы DT_NEEDED. Это происходит, когда в среде "
+"сборки есть разные версии необходимых библиотек. Иногда это делается "
+"намеренно, а иногда нет. Убедитесь, что эти изменения ожидаются. Если это не "
+"так, измените файл спецификации пакета, чтобы гарантировать, что сборка "
+"связана с правильными общими библиотеками."
+
+#: lib/results.h:135
+msgid ""
+"A previously empty file is no longer empty.  Make sure this change is "
+"intended and fix the package spec file if necessary."
+msgstr ""
+"Ранее пустой файл больше не пустой. Убедитесь, что это изменение "
+"преднамеренно, и при необходимости исправьте файл спецификации пакета."
+
+#: lib/results.h:136
+msgid ""
+"A previously non-empty file is now empty.  Make sure this change is intended "
+"and fix the package space file if necessary."
+msgstr ""
+"Ранее непустой файл теперь пуст. Убедитесь, что это изменение преднамеренно, "
+"и при необходимости исправьте файл пространства пакета."
+
+#: lib/results.h:139
+msgid ""
+"Unexpected capabilities were found on the indicated file.  Consult "
+"capabilities(7) and either adjust the files in the package or modify the "
+"capabilities whitelist file in the rpminspect vendor data package.  The "
+"security team may also be of help for this situation."
+msgstr ""
+"В указанном файле обнаружены непредвиденные функции. Обратитесь к функциям "
+"(7) и либо настройте файлы в пакете, либо измените файл белого списка "
```

Comment:
This (7) I think goes with capabilities(7) in which case I would say the capabilities part should not be translated unless there's also a capabilities(7) man page.

Reply:
Updated